### PR TITLE
Convention flag for helicity angles and wigner roatations.

### PR DESCRIPTION
**Pull request by KaiHabermann**
This effects how the boosts in the different systems are done. 

See the corresponding [issue 1](https://github.com/KaiHabermann/decayangle/issues/45) and [issue 2](https://github.com/KaiHabermann/decayangle/issues/44)

### CODE REVIEW

**Comment by KaiHabermann** [comment]
@mmikhasenko Is this what you were thinking off?

**Comment by KaiHabermann** [comment]
Also this is a new feature, so I think we are approaching a version bump to 1.1.0 here

**Comment by KaiHabermann** [comment]
@mmikhasenko could you have a quick look if this is what you meant?

File: src/decayangle/decay_topology.py [hunk 1]
```diff
@@ -292,6 +292,7 @@ def boost(
         target: Union[Node, int],
         momenta: Dict[str, Union[np.array, jnp.array]],
         tol: Optional[float] = None,
+        convention: Literal["helicity", "minus_phi", "canonical"] = "helicity",
```
Comment (mmikhasenko) [inline src/decayangle/decay_topology.py:295]: perhaps, `helicity_minus_phi`...

Comment (mmikhasenko) [reply src/decayangle/decay_topology.py:295]: may be it's fine

**Pull request merged**